pkg/safepath/BUILD.bazel

```diff
@@ -4,6 +4,7 @@ go_library(
     name = "go_default_library",
     srcs = [
         "safepath.go",
+        "safepath_darwin.go",
```

Comment:
So I am sure this is useful, but it seems out of scope for this PR. It is not actually used anywhere else in this PR. Could you make a separate PR that contains this change and the reasoning why it should go in?

Reply:
Removing it, it was just to be able to run unit tests locally 👍 (on macOS)